### PR TITLE
Support timestamped logging even on the terminal

### PR DESCRIPTION
**Pull request by happz**
Pull Request Checklist

* [x] implement the feature
* [x] write the documentation
* [x] extend the test coverage

### CODE REVIEW

**Review by lukaszachy — APPROVED** [review]

**Review by thrix — APPROVED** [review]
LGTM, nice feature!

**Review by janhavlin — APPROVED** [review]

File: tmt/cli.py [hunk 1]
```diff
@@ -185,6 +185,10 @@ def get_command(  # type: ignore[override]
          multiple times.
          """)
 @verbosity_options
+@option(
+    '--log-time-on-terminal',
```
Comment (psss) [inline tmt/cli.py:189]: Could possibly `--log-time` be enough? The current name sounds a little bit long to me...

**Review by psss — APPROVED** [review]
Nice improvement, thanks! Added just a naming suggestion. Not completely sure, rather brainstorming...

File: tmt/cli.py (hunk 1 above)
Comment (happz) [reply tmt/cli.py:189]: Time’s already logged, in the log file. Hence the “terminal”. I don’t mind changing the name, but to what?

Comment (psss) [reply tmt/cli.py:189]: > Time’s already logged, in the log file.

Yeah, my point was that that time is logged in the log file, and always will be, so no future plans to have an option like `--log-time-in-the-log-file`, thus no need to differentiate, so `--log-time` would be just fine with me. Or perhaps something like `--show-time` or `--time-stamps`?

Comment (happz) [reply tmt/cli.py:189]: `--show-time` sounds good, it bears the difference between "showing" logs on the terminal vs. "logging" them into a file.

Comment (happz) [reply tmt/cli.py]: Renamed in e3031cb1d3320d6ef849ac72cb3f59a3cf954db5

Comment (psss) [reply tmt/cli.py]: Good, thanks!

**Pull request merged**